src/parsers/ecs-event-cpio.js

```diff
@@ -14,8 +14,21 @@ const matchingServices = [
 	"thanos-telegraf-v2",
 ];
 
-exports.matches = event =>
-	event.getSource() === "ecs" && matchingServices.includes(event.get("detail.group").slice(8));
+exports.matches = event => {
+	if (event.getSource() === "ecs") {
+		const detailType = event.get("detail-type");
+		var service;
+		if (detailType === "ECS Task State Change") {
+			service = event.get("detail.group").slice(8);
+		}
+		else if (detailType === "ECS Service Action") {
+			const getService = event.parseArn(event.get("resources")).resource;
+			service = getService.slice(8).replace(cluster+'/', '');
+		}
+		return matchingServices.includes(service)
```

Comment:
General comment: We're not worried about null pointer exceptions at all?

Reply:
As I just found out, a null pointer exception in the matches function doesn't crash the whole lambda, it just causes it to move on to the next parser which in this case is the main ecs-event parser.  So it's not a catastrophe if we do get a null pointer.

That said, I'm parsing things out in exactly the same way as the original code does below.  In my earlier PR I essentially forgot to account for the if statement there, so that's what I'm fixing here.